src/components/text/__tests__/index.driver.spec.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import {render} from '@testing-library/react-native';
-import View from '../../view';
-import Text from '../index';
 import {TextDriver} from '../Text.driver.new';
+import {StyleSheet} from 'react-native';
 
 const TEXT_ID = 'text_test_id';
 const TEXT_CONTENT = 'text content';
 
 function WrapperScreenWithText(textProps: {onPress?: jest.Mock} = {}) {
   const {onPress} = textProps;
+  const Text = require('../../text').default;
```

Comment:
Why did you move this here?

Reply:
Because the styling is being set when Text is loaded (in StyleSheet.create) I needed a way to make it load again each time I test. Requiring it there and using jest.isolateModules let me do that because its not being cached this way. This is the Same reason for requiring Constants inside setConstants, because Text uses it it loads it before I can change it so this way by loading it before Text does and changing its values Text uses the appropriate values.